projects/client/RabbitMQ.Client/src/client/impl/ModelBase.cs

```diff
@@ -734,14 +562,6 @@ protected virtual void Dispose(bool disposing)
                 }
                 finally
```

Comment:
Should this `finally` be removed if it's not doing anything?

Reply:
Yeah, it can be removed. I removed setting the event handlers to null since it doesn't do anything (it doesn't unsubscribe listeners).